### PR TITLE
Feat: show cluster displayName instead of name in cluster dashboard

### PR DESCRIPTION
**Pull request by daskyrk**
## What this PR does / why we need it:
show cluster displayName instead of name in cluster dashboard

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | show cluster displayName instead of name in cluster dashboard  |
| 🇨🇳 中文    | 集群总览页使用集群名称替代集群标识作为展示 |

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/dcos/stores/dashboard.ts [hunk 1]
```diff
@@ -78,15 +78,32 @@ const initState: IState = {
   },
 };
 
+const appendDisplayNameToGroupInfo = (groupInfos: ORG_DASHBOARD.IGroupInfo[], clusterList: ORG_CLUSTER.ICluster[]) => {
+  const nameToDisplayName = {};
```
Comment (McDaddy) [inline shell/app/modules/dcos/stores/dashboard.ts:82]: maybe `const nameToDisplayName: Obj<string> = {};` is better

File: shell/app/modules/dcos/stores/dashboard.ts [hunk 2]
```diff
@@ -78,15 +78,32 @@ const initState: IState = {
...
+  map(clusterList, (item) => {
+    nameToDisplayName[item.name] = item.displayName;
+  });
+  return groupInfos.map((group) => {
+    return {
+      ...group,
+      displayName: group.name ? nameToDisplayName[group.name] || group.name : group.name,
```
Comment (McDaddy) [inline shell/app/modules/dcos/stores/dashboard.ts:89]: `displayName: nameToDisplayName[group.name || ''] || group.name,` for short?

File: shell/app/modules/dcos/stores/dashboard.ts [hunk 3]
```diff
@@ -78,15 +78,32 @@ const initState: IState = {
...
-    async getFilterTypes({ call, update }) {
+    async getFilterTypes({ call, update, select }) {
       const { id: orgId, name: orgName } = orgStore.getState((s) => s.currentOrg);
       const clusterList = await call(getClusterList, { orgId });
       if (isEmpty(clusterList)) return;
       const clusterNameString = map(clusterList, (item) => item.name).join();
+
+      const groupInfos = select((s) => s.groupInfos);
+      const newGroupInfos = appendDisplayNameToGroupInfo(groupInfos, clusterList);
+      await update({ groupInfos: newGroupInfos });
```
Comment (McDaddy) [inline shell/app/modules/dcos/stores/dashboard.ts:106]: is this `await` required?

Comment (daskyrk) [reply shell/app/modules/dcos/stores/dashboard.ts:106]: nope

**Review by McDaddy — APPROVED** [review]

**Pull request merged**